### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by riggtravis**
@SChrissndyer I did in fact end up touching a couple of the domains that were failing their tests. Simple stuff. A null pointer exception here and there, and a missing constructor in another place. Were I better human I would have kept closer track and commited more often. For this I am sorry

### CODE REVIEW

File: src/main/java/com/revature/plagex/domains/Classes.java [hunk 1]
```diff
@@ -45,36 +44,44 @@
...
+  /**
+   * @return the assignments
+   */
+  public Set<Assignment> getAssignments() {
+    return assignments;
+  }
 
 	@Override
 	public String toString() {
-		return "Classes [classid=" + classid + ", teacher=" + teacher.getUserID() + "]";
```
Comment (SChrissndyer) [inline src/main/java/com/revature/plagex/domains/Classes.java]: need this other wise there is a loop when calling it

File: src/main/java/com/revature/plagex/domains/Submisions.java [hunk 2]
```diff
@@ -112,8 +112,8 @@ public boolean equals(Object obj) {
 
 	@Override
 	public String toString() {
-		return "Submisions [submissionID=" + submissionID + ", byStudent=" + byStudent.getUserID() + ", forAssignment="
-				+ forAssignment.getAssignmentId() + ", url=" + url + "]";
```
Comment (SChrissndyer) [inline src/main/java/com/revature/plagex/domains/Submisions.java]: same with this if you call the tostring class for this i pulls in other objects which cause a cascading loop

Comment (SChrissndyer) [reply src/main/java/com/revature/plagex/domains/Submisions.java]: @riggtravis

File: src/main/java/com/revature/plagex/domains/Classes.java (hunk 1 above)
Comment (SChrissndyer) [reply src/main/java/com/revature/plagex/domains/Classes.java]: @riggtravis

Comment (SChrissndyer) [reply src/main/java/com/revature/plagex/domains/Classes.java]: nvm on this you can pull the user and the type threw

File: src/main/java/com/revature/plagex/domains/Submisions.java (hunk 2 above)
Comment (SChrissndyer) [reply src/main/java/com/revature/plagex/domains/Submisions.java]: old way ( num,num,num,String)

new( num,obj->(user tostring and userype to string),obj->(assignment toString and class obj-> (class tostring  "which is the other thing you changed")

Comment (SChrissndyer) [reply src/main/java/com/revature/plagex/domains/Submisions.java]: do we want all of this to pull with one toString call?

**Comment by riggtravis** [comment]
Can you forgive me?

**Comment by SChrissndyer** [comment]
looks good to me when did i add usersofthistype?

**Pull request merged**